module-apps/apps-common/ApplicationCommonPopupBlueprints.cpp

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "AppWindow.hpp"
+#include "ApplicationCommon.hpp"
+#include "data/AlarmPopupRequestParams.hpp"
+#include "service-db/Settings.hpp"
+#include "service-db/agents/settings/SystemSettings.hpp"
+#include "popups/data/PopupData.hpp"
+
+namespace app
+{
+    /// TODO should be done with: attachPopups(const std::vector<gui::popup::ID> &popupsList)
+    // TODO what with these popups...
+    // 1. add default popup blueprint?
+    // 2. just do not handle if not registered...
+    // else {
+    //     switchWindow(gui::popup::resolveWindowName(id));
+    // }
+    // Blueprints should be classes
```

Comment:
Hmm you changed that ? i think on some implementation there already were a class?